src/main/groovy/org/edx/jenkins/dsl/AnalyticsConstants.groovy

```diff
@@ -203,4 +203,14 @@ This text may reference other parameters in the task as shell variables, e.g.  $
             }
         }
     }
+
+    public static def opsgenie_heartbeat_publisher = {
+        // No args because the params need to be environment variables to remain hidden.
```

Comment:
Nit: can you update the comment to also explicitly say which environment variables are required.